specification/redisenterprise/resource-manager/Microsoft.Cache/stable/2022-01-01/examples/RedisEnterpriseSkuList.json

```diff
@@ -0,0 +1,608 @@
+{
+    "parameters": {
+      "api-version": "2022-01-01",
+      "subscriptionId": "subid",
+      "location": "westus"
+    },
+    "responses": {
+      "200": {
+        "body": {
+            "value": [
+              {
+                "resourceType": "redisEnterprise",
```

Comment:
Rest APIs are clearer when they use fully qualified resource types like 'Microsoft.Cache/redisEnterprise' instead of just redisEnterprise.